src/components/Dashboard/TodoList.vue

```diff
@@ -100,22 +100,15 @@
                   <ContactInfo class="mt-4 contact-info-warning" />
                 </div>
 
-                <!-- alteration to a BEN in good standing -->
-                <div v-else-if="item.isAlteringToBen && isGoodStanding"
+                <!-- draft alteration to a BEN in good standing -->
+                <div v-else-if="isStatusDraft(item) && item.isAlteringToBen && isGoodStanding"
                   class="todo-subtitle mt-4"
                 >
                   <span>Your business is ready to alter from a {{ item.legalType }} to a BC
                   Benefit Company. Select "Alter Now" to begin your alteration. You will not be able to make
                   any other changes to your business until the alteration is complete.</span>
                 </div>
 
-                <!-- draft correction or alteration -->
-                <div v-else-if="isStatusDraft(item) && (isTypeCorrection(item) || isTypeAlteration(item))"
-                  class="todo-subtitle"
-                >
-                  <span>DRAFT</span>
-                </div>
```

Comment:
Not needed -- will be handled by "draft other" section below.

Reply:
Screenshot (reg user):

![image](https://user-images.githubusercontent.com/10002889/207436440-777d1ae2-848b-4670-8b8e-c4eaf0a833eb.png)

Reply:
I like the screenshots! they help pick out what is relevant.

Reply:
And they show that I tested it :)